recipes/assist-adam-fork/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "assist-adam-fork" %}
+{% set version = "1.1.9.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/assist_adam_fork-{{ version }}.tar.gz
+  sha256: 9d6230ae27b13867a8c48956f1b6c45aa2c945949ef79d007e711f91f621e905
+
+build:
+  number: 2
+  skip: True  # [win]
+  script:
+  - |
+    SUFFIX=$({{ PYTHON }} -c "import distutils.sysconfig as s; print(s.get_config_var('EXT_SUFFIX'))")
+    LIBDIR=$({{ PYTHON }} -c "import distutils.sysconfig as s; print(s.get_python_lib())")
+    L="$LIBDIR/libassist$SUFFIX"
+    echo "BUILDING LIBRARY: $L"
+    {{ PYTHON }} -m pip install . --no-deps -vv
+    if [[ $(uname) == Darwin ]]; then
+      echo "CHECKING LINKED DEPENDENCIES:"
+      otool -L "$L"
+      if ! otool -L "$L" | grep -q "librebound"; then
+        echo "error: librebound not linked"
+        /bin/false
+      else
+        echo "librebound found to be linked, OK!"
+      fi
+      echo "==="
+    fi
```

Comment:
I'm not so sure about this. Maybe @xhochy or @isuruf could take a look?

Reply:
This recipe is taken basically verbatim from the feedstock of the library it is a fork of: https://github.com/conda-forge/assist-feedstock/blob/main/recipe/meta.yaml

Reply:
Checking in if there is anything recommended to be modified. Again, this is in an existing feedstock already.

Reply:
Do let me know if there are any requested modifications.

Reply:
Like I mentioned above, I'm not 100% sure of those lines. Let's ping @xhochy and @isuruf one more time. If they don't answer until next week, ping me again Monday and we'll merge this as-is.

Reply:
Sure thing, thank you.

Reply:
Hi @ocefpag 
Is it okay to merge this in? I'll reiterate that this is using a script identical to one that already has a feedstock.